client/src/components/Main/index.js

```diff
@@ -42,11 +42,12 @@ const Main = () => {
     try {
       const results = await ClientService.events();
       //TODO: call endpoint for invites
-      if (results.success !== undefined && !results.success) {
-        newApiStatus.error = true;
-        newApiStatus.message = results.message;
+      if (results.success) {
+        prepareData(results.events);
       } else {
-        prepareData(results);
+        // default message just in case
+        newApiStatus.error = true;
+        newApiStatus.message = "An error occurred. Please try again later.";
```

Comment:
We could use a const string `An error occurred. Please try again later.` as this is used in multiple places. (Just a comment, might not be worth the extra effort)